addons/tasks/functions/fn_initTaskItems.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: xetra11 
+ *
+ * Initialized the task items for the task module. These items will receive an action
+ *
+ * Arguments:
+ * 1: _taskItems <ARRAY> - task items the action should be applied to
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * Trivial
+ *
+ * Public: No
+ *
+ * Scope: Client
+ */
+
+params [["_taskItems", []]];
+
+if (_taskItems isEqualTo []) exitWith { ERROR("_taskItems was not defined") };
```

Comment:
In my last PR I added a new global ``#define UNDEFINED "undefined"`` . Maybe we could standarize that one and use it with all these checks? Or is the default type of value important?

Reply:
Oh yeah good point. Actually when using a default value you also tell what's the parameters type. So when I say default value is [] and give it a string - the engine will also break out by vanilla errors because the type was wrong.

So for you "undefined" global I think you declared all of those as string types even when you want an array type.

Also there is a problem when you have parameters that give User-Defined names. Think about somebody wants to name his camp/item/marker or whatever just "undefined" (for whatever reason) it won't be possible and he might not understand why.

Reply:
Alright then, I'll keep in consideration this.

Reply:
Remember this conversation? I tested and engine doesn't crash and burn when you give a different data type than the default.